wagtail/admin/templates/wagtailadmin/generic/history_results.html

```diff
@@ -1,6 +1 @@
 {% extends "wagtailadmin/generic/index_results.html" %}
```

Comment:
Unfortunately can't change this to `listing_results.html`, since that template doesn't support the active filters yet. That functionality came for free when it was implemented in the index view, and changing this to `listing_results.html` means we'll lose that.

In addition, the `listing_results.html` template doesn't have the "There are x results" message, so either we need to add that there, or update the tests to not expect it. In any case, that will probably be removed in the future, in favour of showing the number of results in the heading of the first table column like we do for pages.

I opted to leave this in for now.